packages/suite/src/actions/wallet/blockchainActions.ts

```diff
@@ -164,6 +164,20 @@ export const setCustomBackend = (symbol?: string) => async (_: Dispatch, getStat
     return Promise.all(promises);
 };
 
+export const clearCustomBackend = (symbol: string | string[]) => async () => {
+    const coins = typeof symbol === 'string' ? [symbol] : [...new Set(symbol)];
```

Comment:
why using new Set?
shoudnt it be:
`const coins = typeof symbol === 'string' ? [symbol] : symbol;`?

Reply:
It's to dedupe coins. Just in case the same coin is passed multiple times, it will not reconnect X amount of times to blockbook.

Reply:
ah, im still forgetting that custom backends are not grouped by coin